libs/dal/src/repositories/subscriber/subscriber.repository.ts

```diff
@@ -42,7 +42,7 @@ export class SubscriberRepository extends BaseRepository<EnforceEnvironmentQuery
   }
 
   async findBySubscriberId(environmentId: string, subscriberId: string): Promise<SubscriberEntity> {
-    const subscriber = await this.findOne({
+    const subscriber = await super.findOne({
```

Comment:
This part is needed because if we do a query with this.findOne the request will be cached with the subscriberId however the invalidation will be under _id. This one will resolve the invalidation part.

Reply:
As we discussed on our call, this one is problematic and in a sense we did it worse because now we have two requests:
- DB request
- Redis request

So either remove this one at all, or we need to find another way to store based on `subscriberId`

Reply:
Well think about again it the question of worse depends on the amount called to findBySubscriberId and other finds, but I do agree that it is problematic. I will remove it at the moment and will keep in mind going over other options.